src/components/question-format-type.cjsx

```diff
@@ -0,0 +1,60 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+_ = require 'underscore'
+
+{QuestionActions, QuestionStore} = require '../stores/question'
+
+
+PREFIX = 'format'
+TYPES =
+  'multiple-choice' : 'Multiple Choice'
+  'true-false'      : 'True/False'
+  'vocabulary'      : 'Vocabulary'
+  'open-ended'      : 'Open Ended'
+
+QuestionFormatType = React.createClass
+
+
+  update: -> @forceUpdate()
+
+  componentWillMount: ->
+    QuestionStore.addChangeListener(@update)
+
+  componentWillUnmount: ->
+    QuestionStore.removeChangeListener(@update)
+
+  propTypes:
+    questionId: React.PropTypes.number.isRequired
+
+  updateFormat: (ev) ->
+    formats =  QuestionStore.get(@props.questionId).formats or []
+    if ev.target.checked
```

Comment:
Hm, not very easily.  There's not a single checkbox, the `updateFormat` method is called by all the checkboxes `onChange` (one for each format type). 
 
I suppose I could use `_.partial` to create custom callback methods with the type bound to the handler, but it seems like just using the event's values is a bit lighter weight.